catalog/spatial/geocoding/spatial-geocoding-admin-module/src/main/java/org/codice/ddf/spatial/admin/module/service/Geocoding.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.admin.module.service;
+
+import java.lang.management.ManagementFactory;
+
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+
+import org.apache.commons.io.FilenameUtils;
+import org.codice.ddf.spatial.geocoding.GeoEntryExtractionException;
+import org.codice.ddf.spatial.geocoding.GeoEntryExtractor;
+import org.codice.ddf.spatial.geocoding.GeoEntryIndexer;
+import org.codice.ddf.spatial.geocoding.GeoEntryIndexingException;
+import org.codice.ddf.spatial.geocoding.GeoNamesRemoteDownloadException;
+import org.codice.ddf.spatial.geocoding.ProgressCallback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Geocoding implements GeocodingMBean {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Geocoding.class);
+
+    private static final String BASE_DIR = System.getProperty("user.dir") + "/content/store/";
+
+    public GeoEntryExtractor geoEntryExtractor;
+
+    public GeoEntryIndexer geoEntryIndexer;
+
+    private int progress = 0;
+
+    public Geocoding() {
+        registerMbean();
+    }
+
+    private void registerMbean() {
+        ObjectName objectName = null;
+        MBeanServer mBeanServer = null;
+        try {
+            objectName = new ObjectName(Geocoding.class.getName() + ":service=geocoding");
+            mBeanServer = ManagementFactory.getPlatformMBeanServer();
+        } catch (MalformedObjectNameException e) {
+            LOGGER.error("Unable to create Geocoding Configuration MBean.", e);
+        }
+        if (mBeanServer != null) {
```

Comment:
We don't seem to be logging this situation in other implementations.  In my opinion it would be helpful. Should I fix?

https://github.com/codice/ddf/blob/c3b68c2e521070c74330ba3553f65e326be63a76/platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateGenerator.java#L77

https://github.com/codice/ddf/blob/a165f6632e9a09976e1493a80bdfb5a272bf3e97/platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java#L156